app/views/problems/_problem.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.extract! problem, :id, :title, :description, :user_id, :created_at, :updated_at
```

Comment:
Pls remove jbuilder files.

If we want to make API-like JSON render we'll use gem 'ActiveModelSerialiser' for this.